packages/block-library/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Add background color controls for the table block.
 - Add new `RSS` block ([#7966](https://github.com/WordPress/gutenberg/pull/7966)).
 - Add new `Search` block ([#13583](https://github.com/WordPress/gutenberg/pull/13583)).
+- Add vertical alignment controls to `columns` Block ([#13899](https://github.com/WordPress/gutenberg/pull/13899/)) - note: individual (child) `column` Blocks are now selectable within the UI (no longer a "passthrough" Block).
```

Comment:
It will have to be moved to the new `2.4.0 (Unreleased)` section. We should publish new packages to npm tomorrow.

Reply:
Done. Also removed from `editor` changelog